src/order/bounded_lattice.lean

```diff
@@ -280,6 +282,9 @@ begin
   injection H1; injection H2; injection H3; congr'
 end
 
+/-- A `distrib_lattice_bot` is a distributive lattice with a least element. -/
+class distrib_lattice_bot α extends distrib_lattice α, semilattice_inf_bot α, semilattice_sup_bot α
+
 /-- A bounded distributive lattice is exactly what it sounds like. -/
 class bounded_distrib_lattice α extends distrib_lattice α, bounded_lattice α
```

Comment:
Shouldn't you show that all `bounded_distrib_lattice`s are  `distrib_lattice_bot`s? you could maybe just change this to
```suggestion
class bounded_distrib_lattice α extends distrib_lattice_bot α, bounded_lattice α
```
but I haven't tried.

Reply:
Ah yeah sure

Reply:
I'm just wondering whether making all of typeclass go through this new class is a good idea given that it's used for literally 7 lemmas. Maybe we should set up a low priority instance?
I have no clue how we decide which instances should be low priority, so maybe this rant is completely out of point.

Reply:
It doesn't make any difference which approach you use. Adding something to the `extends` clause just tells lean to add the low-priority instance for you.

I would just make the change unless you find it causes timeouts.